File: K47 User Manual/7_miscellany/37_relay/Code/Python/relay.py
#!/usr/bin/env python
import RPi.GPIO as GPIO
import time

RelayPin = 10    

def setup():
	GPIO.setmode(GPIO.BOARD)       # Numbers GPIOs by physical location
	GPIO.setup(RelayPin, GPIO.OUT)

def loop():
	while True:
		print '...relayd on'
		GPIO.output(RelayPin, GPIO.HIGH)
		time.sleep(0.5)
		print 'relay off...'
		GPIO.output(RelayPin, GPIO.LOW)
		time.sleep(0.5)

def destroy():
	GPIO.output(RelayPin, GPIO.LOW)
	GPIO.cleanup()                     # Release resource

if __name__ == '__main__':     		   # Program start from here
	setup()
	try:
		loop()
	except KeyboardInterrupt:  # When 'Ctrl+C' is pressed, the child program destroy() will be  executed.
		destroy()

File: K47 User Manual/7_miscellany/37_relay/Code/C/relay.c

#include <wiringPi.h>
#include <stdio.h>

#define RelayPin      16

int main(void)
{
	if(wiringPiSetup() == -1)
	{
		printf("setup wiringPi failed !");
		return 1; 
	}
	
	pinMode(RelayPin, OUTPUT);
	while(1)
	{
			digitalWrite(RelayPin, HIGH);
			delay(500);
			digitalWrite(RelayPin, LOW);
			delay(500);
	}

	return 0;
}
